docs/diagrams/shion/DeleteSessionSequenceDiagram.puml

```diff
@@ -0,0 +1,85 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":DeleteSessionCommandParser" as DeleteSessionCommandParser LOGIC_COLOR
+participant ":DeleteSessionCommand" as DeleteSessionCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant ":AddressBook" as AddressBook MODEL_COLOR
+participant ":UniqueStudentList" as StudentList MODEL_COLOR
+end box
+
+[-> LogicManager : execute(...)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(...)
+activate AddressBookParser
+
+create DeleteSessionCommandParser
+AddressBookParser -> DeleteSessionCommandParser
+activate DeleteSessionCommandParser
+
+DeleteSessionCommandParser --> AddressBookParser
+deactivate DeleteSessionCommandParser
+
+AddressBookParser -> DeleteSessionCommandParser: parse(...)
+activate DeleteSessionCommandParser
+
+create DeleteSessionCommand
+DeleteSessionCommandParser -> DeleteSessionCommand
+activate DeleteSessionCommand
+
+DeleteSessionCommand --> DeleteSessionCommandParser
+deactivate DeleteSessionCommand
+
+DeleteSessionCommandParser --> AddressBookParser
+deactivate DeleteSessionCommandParser
+
+'Hidden arrow to position the destroy marker slightly below the end of the activation bar.
+DeleteSessionCommandParser -[hidden]-> AddressBookParser
+destroy DeleteSessionCommandParser
+
+AddressBookParser --> LogicManager
+deactivate AddressBookParser
+
+LogicManager -> DeleteSessionCommand : execute()
+activate DeleteSessionCommand
+
+
+DeleteSessionCommand -> Model : deleteSession(name, sessionToAdd)
+activate Model
+
+
+Model --> AddressBook : removeSession(name, index)
+activate AddressBook
+
+AddressBook --> StudentList
```

Comment:
Might want to add the command `deleteSession` in `UniqueStudentList`.

Reply:
Will do so in next commit. when plantUML comes alive

Reply:
nvm not gonna do so now